pkg/containerd/opts/container.go

```diff
@@ -122,7 +122,7 @@ func copyExistingContents(source, destination string) error {
 			return errors.Errorf("volume at %q is not initially empty", destination)
 		}
 
-		if err := chrootarchive.NewArchiver(nil).CopyWithTar(source, destination); err != nil {
+		if err := copy.DirCopy(source, destination, copy.Content, false); err != nil {
```

Comment:
I believe we can remove `copyOwnership()` (just below this), it's already done by `fs.CopyDir()`

Reply:
Done.